examples/src/main/java/com/jagrosh/jdautilities/examples/command/AboutCommand.java

```diff
@@ -15,107 +15,205 @@
-        String author = event.getJDA().getUserById(event.getClient().getOwnerId())==null ? "<@" + event.getClient().getOwnerId()+">" 
-                : event.getJDA().getUserById(event.getClient().getOwnerId()).getName();
-        StringBuilder descr = new StringBuilder().append("Hello! I am **").append(event.getSelfUser().getName()).append("**, ")
-                .append(description).append("\nI ").append(IS_AUTHOR ? "was written in Java" : "am owned").append(" by **")
-                .append(author).append("** using " + JDAUtilitiesInfo.AUTHOR + "'s [Commands Extension](" + JDAUtilitiesInfo.GITHUB + ") (")
-                .append(JDAUtilitiesInfo.VERSION).append(") and the [JDA library](https://github.com/DV8FromTheWorld/JDA) (")
-                .append(JDAInfo.VERSION).append(")\nType `").append(event.getClient().getTextualPrefix()).append(event.getClient().getHelpWord())
-                .append("` to see my commands!").append(join || inv ? invline : "").append("\n\nSome of my features include: ```css");
-        for (String feature : features)
-            descr.append("\n").append(event.getClient().getSuccess().startsWith("<") ? REPLACEMENT_ICON : event.getClient().getSuccess()).append(" ").append(feature);
-        descr.append(" ```");
-        builder.setDescription(descr);
-        if (event.getJDA().getShardInfo() == null)
+        if (inviteLink != null && !inviteLink.isEmpty())
         {
-            builder.addField("Stats", event.getJDA().getGuilds().size() + " servers\n1 shard", true);
-            builder.addField("Users", event.getJDA().getUsers().size() + " unique\n" + event.getJDA().getGuilds().stream().mapToInt(g -> g.getMembers().size()).sum() + " total", true);
-            builder.addField("Channels", event.getJDA().getTextChannels().size() + " Text\n" + event.getJDA().getVoiceChannels().size() + " Voice", true);
+            sb.append("Please [`invite me to your server!`](")
+              .append(inviteLink).append(") ");
         }
-        else
+
+        sb.append("\n\nMy Features include: ```css"); //Use CSS style here for coloring
+        for (Command cmd: commands)
         {
-            builder.addField("Stats", (event.getClient()).getTotalGuilds() + " Servers\nShard " + (event.getJDA().getShardInfo().getShardId() + 1) 
-                    + "/" + event.getJDA().getShardInfo().getShardTotal(), true);
-            builder.addField("This shard", event.getJDA().getUsers().size() + " Users\n" + event.getJDA().getGuilds().size() + " Servers", true);
-            builder.addField("", event.getJDA().getTextChannels().size() + " Text Channels\n" + event.getJDA().getVoiceChannels().size() + " Voice Channels", true);
+            if (cmd.isOwnerCommand()) continue;
+            //You may need to use a Menu Paginator if there are too many Commands
+            //  for 1 MessageEmbed
+            sb.append(cmd.getName()).append("\n");
```

Comment:
This should definitely not be the list of commands, many bots have a huge number of commands and definitely don't want them listed here since they are available in the help. This should remain a list of developer-defined features.